lib/matplotlib/axes/_base.py

```diff
@@ -1690,45 +1686,36 @@ def axis(self, *args, **kwargs):
             else:
                 raise ValueError('Unrecognized string %s to axis; '
                                  'try on or off' % s)
-            xmin, xmax = self.get_xlim()
-            ymin, ymax = self.get_ylim()
-            return xmin, xmax, ymin, ymax
-
-        try:
-            args[0]
-        except IndexError:
-            xmin = kwargs.get('xmin', None)
-            xmax = kwargs.get('xmax', None)
-            auto = False  # turn off autoscaling, unless...
-            if xmin is None and xmax is None:
-                auto = None  # leave autoscaling state alone
-            xmin, xmax = self.set_xlim(xmin, xmax, emit=emit, auto=auto)
-
-            ymin = kwargs.get('ymin', None)
-            ymax = kwargs.get('ymax', None)
-            auto = False  # turn off autoscaling, unless...
-            if ymin is None and ymax is None:
-                auto = None  # leave autoscaling state alone
-            ymin, ymax = self.set_ylim(ymin, ymax, emit=emit, auto=auto)
-            return xmin, xmax, ymin, ymax
-
-        v = args[0]
-        if isinstance(v, bool):
-            if v:
-                self.set_axis_on()
+        else:
+            if len(args) >= 1:
+                if len(args) != 1:
+                    cbook.warn_deprecated(
```

Comment:
Cannot try now, but wasn‘t it possible to pass `axis(xmin, xmax, ymin, ymax)` as four positional arguments, not as a tuple? That‘s not working anymore I think 🤔.

Reply:
No.  They were passable only *as keyword arguments*.  See https://github.com/matplotlib/matplotlib/issues/13668 and linked pr.